tests/html_quotations_tests.js

```diff
@@ -484,6 +486,21 @@ describe("Html Quotations", function () {
       });
     });
 
+    it.skip("should not cut inline blockquote from reply", function () {
```

Comment:
I'm not sure I understand the purpose of the skipped test.

Reply:
I guess it's more for documentation purposes (it's one of the cases I'm trying to fix, so hopefully I'll be able to go back and un-skip it soon). But I could remove it, wdyt?

Reply:
removed the test ✔️